website/docs/d/cas_certificates.html.makrdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_cas_certificates"
+sidebar_current: "docs-alicloud-datasource-cas-certificates"
+description: |-
+    Provides a list of certs available to the user.
+---
+
+# alicloud\_cas\_certificates
+
+This data source provides a list of CAS Certificates in an Alibaba Cloud account according to the specified filters.
+
+## Example Usage
+
+```
+data "alicloud_cas_certificates" "certs" {
+  show_size	= "${var.show_size}"
+  current_page	= "${var.current_page}"
+  lang  = "${var.lang}"
+  output_file = "./tmp.txt"
+}
+
+output "cert" {
+  value = "${alicloud_cas_certificate.certs.*.id}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `output_file` - (Optional) File name where to save data source results (after running `terraform plan`).
+
+## Attributes Reference
+
+The following attributes are exported in addition to the arguments listed above:
+
+* `ids` - A list of domain IDs.
```

Comment:
Please show all of attributes of one certificate.

Reply:
done